### DISCUSSION TITLE
ReadMe

### DISCUSSION THREAD

**Discussion by SananSuleymanov** [coqui-ai/TTS#2683] [General]
Hello, I remember that previously there was a table in the Readme file which shows how to add sounds to speech like smile, speaker cleaning throat and also, information about languages. But right now they are missed. Where can we get detailed information about that?

**Comment by gtvb** [comment]
That seems like a really nice information. Also, is there any info on how I could add pauses between the sentences/words? That would be really nice too. I saw an issue regarding the addition of SSML. Are there any updates on that?

**Comment by erogol** [comment]
I don't remember it. you can check the commit history.
